packages/hash/frontend/src/blocks/page/BlockContextMenu/LoadEntityMenuContent.tsx

```diff
@@ -141,18 +137,32 @@ export const LoadEntityMenuContent: FunctionComponent<
           }}
         />
       </Box>
+      {loading ? (
+        <Box padding={2}>
+          <LoadingSpinner size={16} thickness={4} />
+        </Box>
+      ) : null}
```

Comment:
I think there's also loader in the search input. We can remove that since we are working with this

Reply:
Thanks for spotting @teenoh - I'm going to leave them both for now until we implement a proper design for the feature with filtering of entities etc, because it might change again then